Makefile

```diff
@@ -10,15 +10,28 @@ lint:
 mypy:
 	mypy --ignore-missing-imports $(MODULES)
 
+check_readme:
+	python setup.py check -r -s
```

Comment:
Not important, but I'm not sure if `-r` makes sense since the README is written in Markdown.

Reply:
This is a good point, but it doesn't hurt so I figured I'd leave it.